### ISSUE TITLE
Bad link for 3.8.5 apt install

### ISSUE THREAD

**Issue by flickerfly** [arangodb/arangodb#15689]
I get a 404 when I try to install 3.8.5-1 via apt repo. 3.8.4-1 seems to work just fine.

https://download.arangodb.com/arangodb38/DEBIAN/amd64/arangodb3_3.8.5-1_amd64.deb

**Comment by GhislainAdon** [comment]
Try this [link](https://download.arangodb.com/arangodb38/DEBIAN/amd64/arangodb3-dbg_3.8.5-1_amd64.deb)

**Comment by GhislainAdon** [comment]
you can see all avaible debian package [here](https://download.arangodb.com/arangodb38/DEBIAN/amd64/index.html)

dont forget to close this iisue please

**Comment by flickerfly** [comment]
This is about being able to run `apt-get install arangodb3` which currently breaks because the link to that file breaks. 

With that link I'd have to run apt-get install arangodb3-dbg instead which isn't what I'd prefer to do.

**Comment by GhislainAdon** [comment]
**First add the repository key to apt like this:**

```
curl -OL https://download.arangodb.com/arangodb38/DEBIAN/Release.key
sudo apt-key add - < Release.key
```
**Use apt-get to install arangodb:**

```
echo 'deb https://download.arangodb.com/arangodb38/DEBIAN/ /' | sudo tee /etc/apt/sources.list.d/arangodb.list
sudo apt-get install apt-transport-https
sudo apt-get update
sudo apt-get install arangodb3=3.8.4-1
```
**To install the debug symbols package (not required by default)**

`sudo apt-get install arangodb3-dbg=3.8.4-1`

**Comment by flickerfly** [comment]
Yep, that's what I'm doing now as a work around. I just prefer, in this environment, not to be locked to a specific version because I want to be evaluating compatibility with the latest version. Based on experience with other apt repos, I expect latest stable to be the absence of a specified version.

**Comment by dothebart** [comment]
This specific version has been retracted. A 3.8.5.1 will become available soon. Please go with 3.8.4 for now.

**Comment by Simran-B** [comment]
ArangoDB 3.8.5.1 is out:

`sudo apt-get install arangodb3=3.8.5.1-1`

https://download.arangodb.com/arangodb38/DEBIAN/amd64/arangodb3-client_3.8.5.1-1_amd64.deb

**Comment by maxkernbach** [comment]
Closing this issue as solved.

As mentioned previously, version 3.8.5 has been replaced with 3.8.5.1, which is addressing an [issue in shard synchronization](https://www.arangodb.com/alerts/tech07/).
Please note that the latest available 3.8.x version is 3.8.6 by now.
